tpm2/structures.go

```diff
@@ -119,12 +119,18 @@ func DecodePublic(buf []byte) (Public, error) {
 // precedence. ModulusRaw is used for key templates where the field named
 // "unique" must be a byte array of all zeroes.
 type RSAParams struct {
-	Symmetric  *SymScheme
-	Sign       *SigScheme
-	KeyBits    uint16
-	Exponent   uint32
-	ModulusRaw []byte
-	Modulus    *big.Int
+	Symmetric *SymScheme
+	Sign      *SigScheme
+	KeyBits   uint16
+	// The default Exponent (65537) has two representations; the
+	// zero value, and the value 65537.
+	// If EncodeDefaultExponent is set, an exponent of 65537 will be
```

Comment:
This seems counter-intuitive. Without reading the comment, i'd assume that `EncodeDefaultExponent` means "encode the exponent if it's the default value".
How about `EncodeDefaultExponentAsZero`?

Also, see https://github.com/google/go-tpm/pull/45, which does the opposite - decoding doesn't default to `defaultRSAExponent` when it's 0.

Reply:
I'm happy to rename to `EncodeDefaultExponentAsZero`.

I'm against keeping Exponent in the packed (possibly zero) form - its technical incorrect and potentially breaks code.